<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tcpclient</class>
 <widget class="QDialog" name="tcpclient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tcpclient</string>
  </property>
  <widget class="QPushButton" name="btn_solicitar">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>240</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Solicitar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_salir">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>240</y>
     <width>150</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Salir</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>60</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>120</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_messaje">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>170</y>
     <width>200</width>
     <height>30</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
